docs/docsite/rst/guide_migration.rst

```diff
@@ -28,4 +31,331 @@ Module ``win_iis_website``
 
 Migrated to :ref:`microsoft.iis.website <ansible_collections.microsoft.iis.website_module>`.
 
-To complete.
+community.windows.win_iis_website has been split into two modules:
+  * ansible.microsoft.iis.website.
+  * ansible.microsoft.iis.website_info.
+
+Custom site Parameters from string ($parameters) removed from the module
+win_iis_webbinding was consolidated into microsoft.iis.website
+Parameters bind_ip, bind_port, bind_hostname moved into $bindings ip, port, hostname
+$bindings now include add/set/remove functionality
+Updated from ``Ansible.ModuleUtils.Legacy`` to ``-CSharpUtil Ansible.Basic``
+
+.. code-block:: yaml
```

Comment:
Instead of copying the examples we should show examples of how things were done in the past and how things are now done today with the new module for changed scenarios. For example show how the bindings were specified with the old module and how it is done with the new one. We should also show how the old return values are now retrieved from the `_info` module. Essentially we want to show user's how to migrate from their old code to the new ones.